src/server/external-services/gprofiler/gconvert/index.js

```diff
@@ -133,18 +111,22 @@ const rawValidatorGconvert = ( query, userOptions ) => {
   const defaultOptions = {
     'output': 'mini',
     'organism': 'hsapiens',
-    'target': 'HGNC',
+    'target': COLLECTION_NAMESPACE_HGNC,
     'prefix': 'ENTREZGENE_ACC'
   };
 
-  const form = getForm( query, defaultOptions, userOptions );
-
-  return fetch( GCONVERT_URL, {
+  try{
+    const form = getForm( query, defaultOptions, userOptions );
+    return fetch( GCONVERT_URL, {
       method: 'post',
       body: qs.stringify( form )
-  })
-  .then( response => response.text() )
-  .then( gConvertResponseHandler );
+    })
+    .then( response => response.text() )
+    .then( gConvertResponseHandler );
+  } catch ( err ) {
+    logger.error(`Gprofiler convert query failed - ${ err }`);
+    throw err;
+  }
```

Comment:
You can't mix try-catch with promises.  Either use promises and `.catch()` or use `await` with `try {} catch {}`